pages/index.js

```diff
@@ -2,6 +2,8 @@ import Head from 'next/head';
 import React, { Component } from 'react';
 import { attributes, react as HomeContent } from '../content/home.md';
 import Layout from '../components/Layout';
+import CommunitiesCarousel from "../components/CommunitiesCarousel";
+import {getContentCollection} from "../lib/cms";
```

Comment:
not a big deal but looks like its not formatted, at this point I'm not even sure if prettier is set to format this or not, if it's set up it would be nice to set up your IDE to perhaps auto format on save, then again not 100% necessary since upon committing this should have been formatted in that case due to the precommit hook

I am just a sucker for nice, normalized looking code

Reply:
That's what @nutzaalex means, I think. But this is not your responsibility. The husky pre-commit hook and prettier setup should be taking care of this.

It is possible that the hooks aren't running as expected if you have an older Git version installed. @andreizemba  we require Git `>= 2.13.2` - might that be the issue?

Reply:
Yeah I was pointing to the fact that the mechanism is not working, and not only the quotes, the spacing as well, not a big deal though.

Reply:
hmmm I have
`git --version
git version 2.19.1`

Reply:
I checked the branch today and it's not working for me locally ether, will investigate it further and see if I can get to the bottom of this..

Reply:
Solved by installing prettied plugin in phpstorm :)

Reply:
The main issue however is that the precommit hooks don't work, which should be agnostic of your IDE; having your IDE lint & prettify & code completion etc. tools is for your benefit mostly if it makes sense, while the hooks should format the code to the project's rules no matter what your formatting rules are (you can even have your own IDE settings, maybe you want to use double quotes or ; etc. that can work even though it may be a bit extreme of a case) and upon commiting the hooks should modify and commit the code using the project's rules.

Reply:
It looks that I have some issues with that... need to investigate
```
Can't find yarn in PATH: /Library/Developer/CommandLineTools/usr/libexec/git-core:/usr/bin:/bin:/usr/sbin:/sbin
Skipping pre-commit hook
Can't find yarn in PATH: /Library/Developer/CommandLineTools/usr/libexec/git-core:/usr/bin:/bin:/usr/sbin:/sbin
Skipping commit-msg hook
```